sw_i2c/sample-implementations/linux_user_space/sensirion_sw_i2c_implementation.c

```diff
@@ -77,7 +77,7 @@ static int scl_val_fd;
 static int sda_dir_fd;
 static int sda_val_fd;
 
-static int open_or_exit(const char *path, mode_t mode) {
+static int open_or_exit(const char *path, int mode) {
```

Comment:
I seems `mode` is actually the wrong name for the variable if we look at the declaration of `open`: `int open(const char *pathname, int flags);`

Reply:
Done